### PR TITLE
[agg] Add more flushing counts

### PR DESCRIPTION
**Pull request by ryanhall07**
These extra counters will help understand the amount of work a given
flush task is doing.

1. elements scanned - the number of metric elements scanned by the flush
   task.
2. values processed - the number of values (i.e aggregated timestamps)
   that are dirty and need to be flushed.
3. values expired - the number of values expired and no longer needed.

Additionally noticed we were missing active/inactive elements for timed
metrics, so added those too.

**What this PR does / why we need it**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing and/or backwards incompatible change?**:

```release-note

```

**Does this PR require updating code package or user-facing documentation?**:

```documentation-note

```

### CODE REVIEW

File: src/aggregator/aggregator/aggregator.go [hunk 1]
```diff
@@ -1066,23 +1066,27 @@ type aggregatorTickMetrics struct {
 	duration         tally.Timer
 	standard         tickMetricsForMetricCategory
 	forwarded        tickMetricsForMetricCategory
+	timed            tickMetricsForMetricCategory
 }
 
 func newAggregatorTickMetrics(scope tally.Scope) aggregatorTickMetrics {
 	standardScope := scope.Tagged(map[string]string{"metric-type": "standard"})
 	forwardedScope := scope.Tagged(map[string]string{"metric-type": "forwarded"})
+	timedScope := scope.Tagged(map[string]string{"metric-type": "timed"})
```
Comment (ryanhall07) [inline src/aggregator/aggregator/aggregator.go:1075]: unrelated to flushing, but noticed this type was missing when I was looking at the existing active metrics.

File: src/aggregator/aggregator/elem_base.go [hunk 2]
```diff
@@ -230,63 +237,114 @@ type elemMetrics struct {
...
+		15 * time.Second,
+		20 * time.Second,
+		25 * time.Second,
+		30 * time.Second,
+		35 * time.Second,
+		40 * time.Second,
+		45 * time.Second,
+		60 * time.Second,
+		90 * time.Second,
+		120 * time.Second,
```
Comment (arnikola) [inline src/aggregator/aggregator/elem_base.go:273]: Might be useful to have this go beyond 5m if we expect resolutions above that

**Review by arnikola — APPROVED** [review]
LGTM

File: src/aggregator/aggregator/elem_base.go (hunk 2 above)
Comment (ryanhall07) [reply src/aggregator/aggregator/elem_base.go:273]: true, since it includes jitter.

Comment (ryanhall07) [reply src/aggregator/aggregator/elem_base.go:273]: this makes me realize that the lag time _with_ jitter is pretty useless for high resolution (5m). we won't have enough buckets to understand the actual lag.

**Pull request merged**